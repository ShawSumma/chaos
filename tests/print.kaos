print 77
print "hello world"
print 'hello world'
str hello = "world"
print hello
