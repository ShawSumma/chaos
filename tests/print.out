77
hello world
hello world
world
